packages/story-editor/src/components/panels/document/taxonomies/HierarchicalTermSelector.js

```diff
@@ -221,14 +260,23 @@ function HierarchicalTermSelector({ noParentId = NO_PARENT_VALUE, taxonomy }) {
         onChange={handleClickCategory}
         noOptionsText={taxonomy.labels.not_found}
       />
+      <LinkButton
+        ref={toggleRef}
+        aria-expanded={false}
+        onClick={handleToggleNewCategory}
+        _showAddNewCategory={showAddNewCategory}
```

Comment:
(part 2 of nit re transient props)
```suggestion
        $isVisible={showAddNewCategory}
```